javascript/javascript1/week1/Ez-Namey.js

```diff
@@ -0,0 +1,12 @@
+let firstWords =["Easy", "Awesome", "Corporate", "Nick", "Buy & Go", "Thinknew", "Think Big", "AllyouNeed","One and Only", "All in One"];
+let secondWords =["bujji", "papa", "babu", "laddu", "boondhi", "ammai", "avtar", "asger", "sony", "barathi"];
+var startupName;
+//const randomNumber =firstWords[8];
+//startupName = firstWords(firstWords[8].length);
+const randomNumber = Math.floor(Math.random(firstWords) * 10) + 0;
```

Comment:
How do you think it will differ if you used 2 random indices instead of one; one for each array of words?